src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
+public class User {
+    private Integer id;
+    private String name;
+    private String surname;
+    private Integer phoneNumber;
+    private String mail;
+    private String password;
```

Comment:
Co se týče hesla, je potřeba tam nastavit `@NotNull` a `@Size(min=X, max=Y)`, zde si dej bacha na to, jak dlouhé bude to heslo. A taky mysli na to, že heslo se hashuje.